examples/plot_simple_experiment.py

```diff
@@ -139,15 +149,16 @@
                      'seconds to push the buttons in the order that the tones '
                      'played in. Press one of the buttons to begin.'
                      ''.format(len(mass_trial_order), max_resp_time),
-                     live_keys=live_keys)
+                     live_keys=live_keys, max_wait=max_wait)
     ec.load_buffer(concat_wavs)
     ec.identify_trial(ec_id='multi-tone', ttl_id=[0, 1])
     ec.write_data_line('multi-tone trial', [x + 1 for x in mass_trial_order])
     ec.start_stimulus()
-    ec.wait_secs(len(concat_wavs) / float(ec.stim_fs))
+    ec.wait_secs(len(concat_wavs) / float(ec.stim_fs) if not building_doc else
+                 0)
     ec.screen_text('Go!', wrap=False)
     ec.flip()
-    pressed = ec.wait_for_presses(max_resp_time + 1, min_resp_time,
+    pressed = ec.wait_for_presses(long_resp_time, min_resp_time,
```

Comment:
this is still not going to work.  It won't error out, but the comparison `press == ans` a couple lines down will always yield 0 out of 5, because it will be comparing `tuple`s to `int`s